### ISSUE TITLE
pyodide-pr-bot integration

### ISSUE THREAD

**Issue by ryanking13** [pyodide/pyodide#4350]
Related: https://github.com/pyodide/pyodide/pull/4345

@agoose77 created a very useful tool [`pyodide-pr-bot`](https://github.com/agoose77/pyodide-pr-bot) which opens a PR to update pyodide package recipes automatically. This issue is to track the progress of integrating the bot into Pyodide repository.

## Things need to be done

- [ ] transfter `pyodide-pr-bot` to Pyodide organization
  - If @agoose77 transfer the ownership to me, then I will transfer it to Pyodide organization.

## General discussion topics

1. How do we register packages that can be updated?

For now, there is a json file inside pyodide-pr-bot, but we can think of adding some metadata in `meta.yaml`, such as:

```yaml
extra:
  recipe-maintainers:
    - <github_username>
  auto-update: true
  ...other keys...
```

2. How to handle if we start to "split" recipes?

Currently, all of Pyodide's recipes are in the pyodide repository, but this is not scalable and I'm exploring ways to address this.

A start would be to move recipes to other repositories (which is what I am doing [here](https://github.com/ryanking13/pyodide-recipes-mirror)), and in the long run, recipes may be distributed across multiple repositories, similar to conda-forge. So pyodide-pr-bot might need to handle those cases later.

**Comment by agoose77** [comment]
@ryanking13 I've started the transfer to you, but it probably will take some time — I'd tried transferring from `agoose77` to `pyodide-pr-bot` first, but it never completed. I suspect a name conflict, which I've resolved.

**Comment by ryanking13** [comment]
Thanks! I just transfered the repository to https://github.com/pyodide/pr-integrator. Please let me know if there is any issue.

**Comment by agoose77** [comment]
Nice!

**Comment by agoose77** [comment]
@ryanking13 perhaps you'd now like to have the credentials for the GitHub bot account?

**Comment by ryanking13** [comment]
Yeah, probably. I wish there was a good way to manage bot accounts other than sharing the raw password, but I'm not sure.
The required GitHub token seems already registered in pr-integrator repository, so maybe we can ask for the credential when we need.

**Comment by agoose77** [comment]
Yeah agreed. Well, I created a whole new email for it, so I'm fine to just giving you the creds from my password manager.

**Comment by ryanking13** [comment]
I see. Then, could you send the bot account's credentials to my email (def6488@gmail.com)?
